packages/component/src/Tracker.js

```diff
@@ -0,0 +1,64 @@
+import { useEffect } from 'react';
+
+import { checkSupport as supportWorker } from './Utils/downscaleImageToDataURLUsingWorker';
+import { speechSynthesis } from './Speech/BypassSpeechSynthesisPonyfill';
+import useDebugDeps from './hooks/internal/useDebugDeps';
+import useLanguage from './hooks/useLanguage';
+import useTrackDimension from './hooks/useTrackDimension';
+import useTrackEvent from './hooks/useTrackEvent';
+import useWebSpeechPonyfill from './hooks/useWebSpeechPonyfill';
+
+const Tracker = () => {
```

Comment:
Why is `Tracker` a headless component instead of a hook?

Reply:
Dang, you are right.

Reply:
Oh, because we can't easily do it without another layer of component.

But I agree we should move headless component to hooks as much as we could.

I am going to move the "business logic" of `<Tracker>` as a hook. And later when we clean up things, e.g. clean up `<Composer>` or switch input mode cleaning up `<Dictation>`. We move the bogus component.

```js
  return (
    <WebChatUIContext.Provider value={context}>
      <SayComposer ponyfill={webSpeechPonyfill}>
        {typeof children === 'function' ? children(context) : children}
      </SayComposer>
      <Dictation onError={dictationOnError} />
      {onTelemetry && <Tracker />}
    </WebChatUIContext.Provider>
  );
```

Reply:
Oh...